apps/cowswap-frontend/src/common/hooks/orderProgressBarV2/types.ts

```diff
@@ -15,8 +15,8 @@ export type OrdersProgressBarState = Record<string, OrderProgressBarState>
 
 export type OrdersProgressBarCountdown = Record<string, number | null>
 
-type happyPath = 'initial' | 'solving' | 'solved' | 'executing' | 'finished'
-type errorFlow = 'nextBatch' | 'delayed' | 'unfillable' | 'submissionFailed'
+type happyPath = 'initial' | 'solving' | 'executing' | 'finished'
+type errorFlow = 'delayed' | 'solved' | 'unfillable' | 'submissionFailed'
```

Comment:
not coming from this PR, but shouldn't the types be capitalised?

Reply:
Nope

Reply:
They are sort of inner logical division. The exported one is the combination of all of them.